uelc/main/models.py

```diff
@@ -188,6 +174,22 @@ def __unicode__(self):
     def display_name(self):
         return '%s' % (self.name)
 
+    def _get_cohorts(self):
+        cohorts = Cohort.objects.filter(case_cohort=self.id)
+        if not cohorts:
+            return None
```

Comment:
I don't know if this check is necessary. I'd think you can just do `return Cohort.objects.filter(case_cohort=self.id)`, which will return an empty array when there's no cohorts matching the query. Is there a reason you'd like to return `None`?